### DISCUSSION TITLE
🔗 Add Custom icon for WooCommerce Notices widget (Related to #25710)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#25709] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

As you know, WooCommerce notifications have icon by default. Currently, there is no option to change the default icons. So it would be really great if you add the option to choose a custom icon for WooCommerce Notices.

### Describe the solution you'd like

Ability to choose a custom icon for WooCommerce Notices.

![a](https://user-images.githubusercontent.com/88034902/213911463-f1f07e1f-36e2-4491-a0a9-cc2ae95c0cdf.png)

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #21092

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by nicholaszein** [comment]
This Request was mentioned in the following issues and discussions:

- #25710
